app/controllers/renewal/RenewalProgressController.scala

```diff
@@ -59,12 +59,13 @@ class RenewalProgressController @Inject()
               cache <- OptionT(dataCacheConnector.fetchAll)
               businessMatching <- OptionT.fromOption[Future](cache.getEntry[BusinessMatching](BusinessMatching.key))
             } yield {
-
+              val businessName = businessMatching.reviewDetails.map(r => r.businessName).getOrElse("")
+              val activities = businessMatching.activities.fold(Seq.empty[String])(_.businessActivities.map(_.getMessage()).toSeq)
```

Comment:
Do we need the toSeq call? I suspect we do, but just to clarify.

Reply:
It does currently, the template for services_sidebar was pre-existing and takes a Seq.

Reply:
A most acceptable answer